util/minetestmapper.py

```diff
@@ -190,8 +190,77 @@ def usage():
         draworigin = True
     elif o == "--drawunderground":
         drawunderground = True
+    elif o == "--geometry":
+        geometry_string = a
+    elif o == "--region":
+        region_string = a
     else:
         assert False, "unhandled option"
+nonchunky_xmin=-1500
+nonchunky_xmax=1500
+nonchunky_zmin=-1500
+nonchunky_zmax=1500
+
+if geometry_string is not None:
+    parts = geometry_string.split("+")
+    if len(parts) == 3:
+        coords_string, width_string, height_string = parts
+        coords = coords_string.split(":")
+        if len(coords)==2:
+            x_string, z_string = coords
+            x = int(x_string)
+            z = int(z_string)
```

Comment:
What about a list comprehension or lambda here?
`x, z = [int(foo) for foo in coords]`